discovery-provider/src/tasks/entity_manager/entity_manager.py

```diff
@@ -267,6 +257,33 @@ def entity_manager_update(
                     ):
                         revoke_grant(params)
 
+                    # add processed metadata to cid_metadata dicts to batch save to cid_data table
+                    # later
+                    if expect_cid_metadata_json(
+                        params.metadata, params.action, params.entity_type
+                    ):
+                        if getattr(params, "updated_metadata", None):
```

Comment:
this merging is backwards compatible right?

Reply:
Yes I believe so! It will pull the previous metadata from the table and merge in the new metadata using the `|` operator. Can double check backwards compatibility once on staging